src/sentry/api/endpoints/project_ownership.py

```diff
@@ -20,11 +26,29 @@
 HIGHER_MAX_RAW_LENGTH = 200_000
 
 
-class ProjectOwnershipSerializer(serializers.Serializer):
-    raw = serializers.CharField(allow_blank=True)
-    fallthrough = serializers.BooleanField()
-    autoAssignment = serializers.CharField(allow_blank=False)
-    codeownersAutoSync = serializers.BooleanField(default=True)
+class ProjectOwnershipRequestSerializer(serializers.Serializer):
+    raw = serializers.CharField(
+        required=False,
+        allow_blank=True,
+        help_text="Raw input for ownership configuration. Check out [Ownership Rules Documentation](/product/issues/ownership-rules/) to learn more.",
+    )
+    fallthrough = serializers.BooleanField(
+        required=False,
+        help_text="Indicate if there is no match on explicit rules, to fall through and make everyone an implicit owner.",
```

Comment:
nit: I found this wording a little bit confusing. Could word it similar to the text for codeownersAutoSync, ie. "Set to `True` to make everyone an implicit owner if there is no match on explicit rules."